### PR TITLE
SYS-177 Enable mysql slow_query_log

### PR DESCRIPTION
**Pull request by zazabe**

### CODE REVIEW

**Comment by zazabe** [comment]
please hold, ill add logrotate for slow queries

**Comment by ppp0** [comment]
sgtm! just because I don't like things that grow indefinitely

**Comment by zazabe** [comment]
please review

File: modules/mysql/spec/server/spec.rb [hunk 1]
```diff
@@ -25,4 +25,20 @@
...
+    end
+  end
+
+  describe file('/var/log/mysql-slow-query.log') do
+    it "owned by user mysql" do
+      expect(subject).to be_owned_by('mysql')
+    end
+  end
+
+  describe command('mysql -e "select sleep (1.1);" && cat /var/log/mysql-slow-query.log' ) do
```
Comment (ppp0) [inline modules/mysql/spec/server/spec.rb]: I like

**Pull request merged**